src/App.tsx

```diff
@@ -17,18 +48,35 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelect={setSelect}
+                selectValue={select}
+                query={query}
+                setQuery={setQuery}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {visibleToDos.length || query
+                ? (
+                  <TodoList
+                    todos={visibleToDos}
+                    chooseTodo={setSelectedTodo}
```

Comment:
```suggestion
                    onTodoSelect={setSelectedTodo}
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364

Reply:
and it's better to create handle functions

Reply:
Thanks for this link! Now It's become much clearer!